src/platforms/SecretText/TextCardButton.tsx

```diff
@@ -35,6 +38,8 @@ const TextCardButton = ({ platform }: Props) => {
     typeof rolePlatform?.capacity === "number" &&
     rolePlatform?.capacity === rolePlatform?.claimedCount
       ? "All available rewards have already been claimed"
+      : claimed
```

Comment:
If the user has claimed the reward, we shouldn't display the tooltip, so we don't need these two lines here